[2025-07-05T06:10:31.139+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-05T06:10:31.308+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_task_group.section_1.task_2 scheduled__2025-07-04T00:00:00+00:00 [queued]>
[2025-07-05T06:10:31.364+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_task_group.section_1.task_2 scheduled__2025-07-04T00:00:00+00:00 [queued]>
[2025-07-05T06:10:31.364+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-05T06:10:31.460+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): section_1.task_2> on 2025-07-04 00:00:00+00:00
[2025-07-05T06:10:31.498+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=518) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-05T06:10:31.498+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_task_group', 'section_1.task_2', 'scheduled__2025-07-04T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_task_group.py', '--cfg-path', '/tmp/tmp50okquxm']
[2025-07-05T06:10:31.500+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask section_1.task_2
[2025-07-05T06:10:31.499+0000] {standard_task_runner.py:63} INFO - Started process 629 to run task
[2025-07-05T06:10:31.734+0000] {task_command.py:426} INFO - Running <TaskInstance: example_task_group.section_1.task_2 scheduled__2025-07-04T00:00:00+00:00 [running]> on host b39724c39614
[2025-07-05T06:10:32.005+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_task_group' AIRFLOW_CTX_TASK_ID='section_1.task_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-04T00:00:00+00:00'
[2025-07-05T06:10:32.006+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-05T06:10:32.007+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-05T06:10:32.022+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 1']
[2025-07-05T06:10:32.145+0000] {subprocess.py:86} INFO - Output:
[2025-07-05T06:10:32.177+0000] {subprocess.py:93} INFO - 1
[2025-07-05T06:10:32.181+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-05T06:10:32.182+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-05T06:10:32.238+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_task_group, task_id=section_1.task_2, run_id=scheduled__2025-07-04T00:00:00+00:00, execution_date=20250704T000000, start_date=20250705T061031, end_date=20250705T061032
[2025-07-05T06:10:32.287+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-05T06:10:32.392+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-05T06:10:32.394+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
